devel/bitwise/Portfile

```diff
@@ -0,0 +1,27 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+
+github.setup        mellowcandle bitwise v0.30
+github.tarball_from releases
+
+name                bitwise
+categories          devel
+license             GPL-3
+maintainers         {@herrbischoff herrbischoff.com:marcel} \
+                    openmaintainer
+description         Terminal based bit manipulator in ncurses.
+long_description    Bitwise is a multi base interactive calculator supporting\
+                    dynamic base convertion and bit manipulation. It's a handy\
+                    tool for low level hackers, kernel developers and device\
+                    driver developers.
+platforms           darwin
+homepage            https://github.com/mellowcandle/bitwise
+
+checksums           rmd160  69860c19945594af5026c1abdbbee7f83c26457f \
+                    sha256  4cc259c9d5d6aa1ba4459d3b18b983da005361a9af57ced9f74970ee9af681c6 \
+                    size    112084
+
+depends_lib         lib:readline:readline \
```

Comment:
Normally we just use a port dependency here. Why are you using a library dependency for readline?

Reply:
Probably an oversight on my part. I'll change it to a port.

Reply:
Done.